bin/launch-cityzen-server.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+
+require('dotenv').config()
+const child_process = require('child_process')
+
+console.log('starting cityzen server...')
+
+let { CITYZEN_SERVER_AUTOLAUNCH_BRANCH, CITYZEN_SERVER_AUTOLAUNCH_URL } = process.env
+
+CITYZEN_SERVER_AUTOLAUNCH_BRANCH = CITYZEN_SERVER_AUTOLAUNCH_BRANCH || 'master'
+CITYZEN_SERVER_AUTOLAUNCH_URL = CITYZEN_SERVER_AUTOLAUNCH_URL || `git://github.com/codeforlansing/cityzen-server.git#${CITYZEN_SERVER_AUTOLAUNCH_BRANCH}`
+
+child_process.spawn('npx', ['--ignore-existing=false', CITYZEN_SERVER_AUTOLAUNCH_URL], { stdio: 'inherit' })
```

Comment:
Expanding on this, I think it would be helpful to offer a way to pass command line arguments to the server. For example, to start the server using Trello, I run the command `npm start -- --config trello.json`.